Loadouts/msv_ak74_emr.hpp

```diff
@@ -0,0 +1,312 @@
+  magazines[] += {AT_MAG};
+};
+class potato_msv_sm: potato_msv_rifleman {// Medic
+  vest[] = {"rhs_6b23_digi_medic"};
+  magazines[] = {CARBINE_MAG,MEDIC_GRENADES};
+  backpackItems[] = {MEDIC_MEDICAL};
+};
+class potato_msv_marksman: potato_msv_rifleman {// Squad Marksman
+  weapons[] = {MARKSMAN};
+  magazines[] = {MARKSMAN_MAG};
+  attachments[] = {"rhs_acc_pso1m21"};
+  opticChoices[] = {"rhs_acc_pso1m21","rhs_acc_1pn93_1"};
+};
+class Fic_Spotter: potato_msv_rifleman {
+  linkedItems[] += {RANGE_FINDER};
+};
+class potato_msv_mmgg: potato_msv_AR {// MMG
+  backpack[] = {"rhs_assault_umbts"};
+  weapons[] = {MMG};
+  magazines[] = {MMG_MAG,PISTOL_MAG,BASE_GRENADES};
+};
+class potato_msv_mmgag: Fic_Spotter {// MMG Spotter/Ammo Bearer
+  backpack[] = {"rhs_assault_umbts"};
+  backpackItems[] += {MMG_MAG};
+};
+class potato_msv_matg: potato_msv_rifleman {// MAT Gunner
+  backpack[] = {"rhs_rpg_empty"};
+  backpackItems[] = {};
+  magazines[] += {MAT_MAG};
+  items[] += {BASE_MEDICAL};
+  launchers[] = {MAT};
+  secondaryAttachments[] = {MAT_OPTIC};
+};
+class potato_msv_matag: Fic_Spotter {// MAT Spotter/Ammo Bearer
+  backpack[] = {"rhs_rpg_empty"};
+  backpackItems[] = {};
+  magazines[] += {MAT_MAG2};
+  items[] += {BASE_MEDICAL};
+};
+class potato_msv_msamg: potato_msv_rifleman {// SAM Gunner
+  SAM_GEAR("B_Carryall_oli", SAM_MAG)
+  launchers[] = {SAM};
+};
+class potato_msv_msamag: Fic_Spotter {// SAM Spotter/Ammo Bearer
+  SAM_GEAR("B_Carryall_oli", SAM_MAG)
+};
+class potato_msv_msaml: potato_msv_msamag {// SAM Leader
+  items[] += {LEADER_TOOLS};
+  linkedItems[] += {LEADER_LINKED,RADIO_LR};
+};
+class potato_msv_mtrl: Fic_Spotter { // Mortar Leader
+  items[] += {LEADER_TOOLS};
+  linkedItems[] += {LEADER_LINKED,RANGE_FINDER,RADIO_LR};
+};
+class potato_msv_mtrg: potato_msv_rifleman {// Mortar Gunner
+  MORTAR_GEAR("O_Mortar_01_weapon_F")
+};
+class potato_msv_mtrag: Fic_Spotter {// Assistant Mortar
+  MORTAR_GEAR("O_Mortar_01_support_F")
+};
+class spotter_F: Fic_Spotter {// Spotter
+  weapons[] = {SPOTTER};
+  magazines[] = {SPOTTER_MAG,BASE_GRENADES};
+  items[] += {RADIO_MR,"ACE_ATragMX","ACE_Kestrel4500","ACE_RangeCard"};
+  linkedItems[] += {LEADER_LINKED};
+  attachments[] = {"rhs_acc_pso1m2"};
+};
+class potato_msv_sniper: spotter_F {// Sniper
+  weapons[] = {SNIPER};
+  magazines[] = {SNIPER_MAG,BASE_GRENADES};
+  items[] = {BASE_TOOLS,"ACE_RangeCard"};
+  linkedItems[] = {BASE_LINKED,LEADER_LINKED};
+  attachments[] = {"rhs_acc_pso1m21"};
+  opticChoices[] = {"rhs_acc_pso1m21","rhs_acc_1pn93_1"};
+};
+class potato_msv_pilot {// Pilot
+  uniform[] = {"rhs_uniform_df15"};
+  vest[] = {"V_TacVest_blk"};
+  headgear[] = {"rhs_zsh7a_mike"};
+  backpack[] = {"B_AssaultPack_rgr"};
+  weapons[] = {SMG};
+  magazines[] = {SMG_MAG,CREW_GRENADES};
+  backpackItems[] = {KEY_EAST,RADIO_LR};
+  items[] = {BASE_MEDICAL,BASE_TOOLS,LEADER_TOOLS,RADIO_MR,"rhsusf_ANPVS_15"}; // Theres no double tube NVGs for pilots in RHS yet
```

Comment:
It looks really stupid when they're hanging off the helmet so I think it's better to just put it in the inventory, that way the pilot only puts it on if he has to.